react-native/redux/custom-redux-form-field.md

```diff
@@ -0,0 +1,104 @@
+# [MO] How to use redux-form with custom fields (~10min)
+
+## Owner : [Darius Afchar](https://github.com/DariusAf)
+
+*Why would you want to do that ?*
+
+Redux-form is great and can automate a lot of things as form-validation, focus events, storage synchronisation... But it can be a bit tricky if you want to add a custom component that includes many fields whose values have to be aggregated, or an external component, for instance a date-picker.
+
+## Prerequisities
+
+A react project with a redux store, obviously.
```

Comment:
Check point an no 'obviously'. Potentially, the reader have no idea what a redux store is.

That would be better:
```
- a react project : create one with create-react-app (link)
- redux installed : link to a good but simple step to install redux
```